File: backend/api/src/main/scala/com/github/ai/split/api/response/PutGroupResponse.scala
package com.github.ai.split.api.response

import com.github.ai.split.api.GroupDto
import zio.json.{DeriveJsonEncoder, JsonEncoder}

case class PutGroupResponse(
  group: GroupDto
)

object PutGroupResponse {
  implicit val encoder: JsonEncoder[PutGroupResponse] = DeriveJsonEncoder.gen[PutGroupResponse]
}

File: backend/api/src/main/java/com/github/ai/split/api/response/PutGroupResponse.java
package com.github.ai.split.api.response;

import com.github.ai.split.api.GroupDto;
public class PutGroupResponse {

    public GroupDto group = new GroupDto();

    public PutGroupResponse() {}

    public PutGroupResponse(GroupDto group) {
        this.group = group != null ? group : new GroupDto();
    }
}